src/SDL_sound.h

```diff
@@ -50,10 +50,10 @@
 #ifndef _INCLUDE_SDL_SOUND_H_
 #define _INCLUDE_SDL_SOUND_H_
 
-#include "SDL.h"
+#include <SDL3/SDL.h>
 
```

Comment:
I'd rename the include guard to not start with `_`, and move this header to `include/SDL3_sound/SDL_sound.h`.
CMake and Android.mk should be updated accordingly.

Reply:
Leaving this for later for now